jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/IteratingCallback.java

```diff
@@ -229,10 +229,10 @@ public void iterate()
             }
         }
         if (process)
-            processing();
+            processing(true);
     }
 
-    private void processing()
+    private void processing(boolean processing)
```

Comment:
Maybe rename that `processing` boolean into `doProcess`? Otherwise having a method that accepts a boolean with the exact same name gets confusing.